uiowa/uiowa
# [Brand icons] Brand icon media type

Resolves https://github.com/uiowa/uiowa/issues/8211. 



# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /admin/content/media
```
1. Sync site
2. Uncomment update hook
3. `ddev drush @sandbox.local updb`
4.  Go to media page to confirm icons were imported, compare imported data to icons.json. 
5. Go to `/admin/structure/block-content/manage/uiowa_card/fields/block_content.uiowa_card.field_uiowa_card_image` and set "Brand Icon" as a media type. 
6. Create a card block and add an icon as an image. 

## Still to-do
- [ ] Remove Brand Icon from `/admin/content/media-grid` and `/admin/content/media views`
